appinventor/appengine/src/com/google/appinventor/client/DesignToolbar.java

```diff
@@ -545,4 +566,5 @@ public void setTutorialToggleVisible(boolean value) {
     }
   }
 
-}
+
```

Comment:
The newline should be after the `}`. Since no semantic change has occurred here, GitHub won't show this if you update the file correctly.